rpc/params/caas.go

```diff
@@ -60,6 +60,7 @@ type CAASApplicationProvisioningInfo struct {
 	Trust                bool                         `json:"trust,omitempty"`
 	Scale                int                          `json:"scale,omitempty"`
 	Error                *Error                       `json:"error,omitempty"`
+	StorageUniqueID      string                       `json:"storage-unique-id,omitempty"`
```

Comment:
This will never be empty now right? 
We should keep Error as the last attribute, so move this up higher.